tests/test_telemetry.py

```diff
@@ -0,0 +1,196 @@
+import asyncio
+import json
+from pathlib import Path
+import uuid
+
+from _pytest.monkeypatch import MonkeyPatch
+import jsonschema
+from mock import Mock
+import pytest
+
+from rasa import telemetry
+import rasa.constants
+from rasa.shared.importers.importer import TrainingDataImporter
+from tests.conftest import DEFAULT_CONFIG_PATH
+
+TELEMETRY_TEST_USER = uuid.uuid4().hex
+TELEMETRY_TEST_KEY = uuid.uuid4().hex
+TELEMETRY_EVENTS_JSON = "docs/docs/telemetry/events.json"
+
+
+@pytest.fixture(autouse=True)
+def patch_global_config_path(monkeypatch: MonkeyPatch, tmp_path: Path):
+    """Ensure we use a unique config path for each test to avoid tests influencing
+    each other."""
+    default_location = rasa.constants.GLOBAL_USER_CONFIG_PATH
```

Comment:
why are we not monkeypatching this?

Reply:
didn't work, but I didn't bother to investigate why

Reply:
mhm, ok. then let's at least add the missing type annotation and drop the unused `monkeypatch` param

Reply:
did that already :+1: